themes/src/main/resources/theme/keycloak.v2/account/src/app/content/signingin-page/SigningInPage.tsx

```diff
@@ -348,14 +360,25 @@ class CredentialAction extends React.Component<CredentialActionProps> {
         if (this.props.removeable) {
             const userLabel: string = this.props.credential.userLabel;
             return (
-                <DataListAction aria-labelledby='foo' aria-label='foo action' id={'removeAction-' + this.props.credential.id }>
-                    <ContinueCancelModal buttonTitle='remove'
-                                        buttonId={`${SigningInPage.credElementId(this.props.credential.type, this.props.credential.id, 'remove')}`}
-                                        modalTitle={Msg.localize('removeCred', [userLabel])}
-                                        modalMessage={Msg.localize('stopUsingCred', [userLabel])}
-                                        onContinue={() => this.props.credRemover(this.props.credential.id, userLabel)}
-                                            />
-                </DataListAction>
+		<>
+			<DataListAction aria-labelledby='foo' aria-label='foo action' id={'removeAction-' + this.props.credential.id }>
+			    <ContinueCancelModal buttonTitle='remove'
+						buttonId={`${SigningInPage.credElementId(this.props.credential.type, this.props.credential.id, 'remove')}`}
+						modalTitle={Msg.localize('removeCred', [userLabel])}
+						modalMessage={Msg.localize('stopUsingCred', [userLabel])}
+						onContinue={() => this.props.credRemover(this.props.credential.id, userLabel)}
+						    />
+			</DataListAction>
+			<DataListAction aria-labelledby='foo' aria-label='foo action' id={'setDefaultAction-' + this.props.credential.id } >
```

Comment:
Set default button should probably be before the remove button.

Reply:
That sounds logical to me